facebook/react-native
# Remove ActivityIndicator from ScrollView snapshot tests

`ActivityIndicator` was not consistent when running snapshot tests so I removed it.

From diff:
![image](https://user-images.githubusercontent.com/7029942/50274094-9c662700-043c-11e9-9cac-ccc1b69695c1.png)

Changelog:
----------

[iOS] [FIXED] - Fixed ScrollViewExample snapshot tests


Test Plan:
----------

1. Run `yarn` at the repository root.
2. Open `RNTester/RNTester.xcodeproj` on Xcode 10.1.0.
3. Build RNTester.
4. Run tests (CMD+U) on an iPhone 6s simulator (check only `RNTesterSnapshotTests`).

Fixes #22696
